demo/config/bulk-source.properties

```diff
@@ -0,0 +1,5 @@
+name=cassandra-source-bulk
```

Comment:
This is a sample/example app vs a demo right? I might change the name of the dir and any supporting documentation to 'examples', that way if someone wants to add another in the future...